src/parser/FStar.Parser.Dep.fs

```diff
@@ -126,27 +126,30 @@ type dep_node = {
 }
 type dependence_graph = //maps file names to the modules it depends on
      | Deps of smap<dep_node> //(dependences * color)>
+type parsing_data = list<dependence> * list<dependence> * bool
```

Comment:
a comment here describing the three fields would be useful.

Reply:
i guess it just reusing this comment, or moving it here would be good

```
   list<dependence> //direct dependences of filename
 * list<dependence> //additional "roots" that should be scanned
                    //i.e. implementations of interfaces in the first list
 * bool             //interface that contains an `inline_for_extraction```

Reply:
Fixed in https://github.com/FStarLang/FStar/commit/f5ba9096b86f8a0cf4243c587cb2ef8f1ccbb140#diff-bb2dbf1990bd066ee7fcfad0a2fd8b3f.